src/command_modules/azure-cli-keyvault/HISTORY.rst

```diff
@@ -2,6 +2,15 @@
 
 Release History
 ===============
+2.0.4 (2017-05-08)
+++++++++++++++++++++
+
+* Updated to use version 0.3.0 of azure-keyvault SDK
+
+2.0.3 (2017-05-05)
+++++++++++++++++++++
+
+* Minor fixes.
```

Comment:
This section is for customer facing changes only. Since the command surface area hasn't changed, nothing is required (or desired) here.